src/ui/server/controllers/insurance/policy/broker-addresses/index.ts

```diff
@@ -103,6 +104,24 @@ export const get = async (req: Request, res: Response) => {
 
     const { addresses } = response;
 
+    /**
+     * If only one address is found - no need to ask the user to select an address. Therefore:
+     * 1) Generate an address to save.
+     * 2) Send to the API.
+     * 3) Redirect to the next part of the flow.
+     */
+    if (addresses.length === 1) {
```

Comment:
Is there a reason for a `1` length check?

Reply:
Yes, it's in the PR description and above documentation (line 108) 😛 

If only 1 address is returned, we need to redirect a user to a different URL (to skip 1x step). Otherwise a user is redirected to a form where there is only possible option to select.